cypress/integration/userCanSeeMeals.feature.js

```diff
@@ -0,0 +1,18 @@
+describe('User can see Meals', () => {
+    before(function () {
+        cy.visit('/')
```

Comment:
you are not stubbing out the requests that fetches the product data, that is why it is failing on semaphore

Reply:
Fixed this now we think!